src/commands/promote.js

```diff
@@ -10,15 +10,29 @@ const promote = (context, version) => {
   let appId = 0;
 
   return utils.checkCredentials()
-    .then(() => utils.getLinkedApp())
-    .then((app) => {
+    .then(() => Promise.all([
+      utils.getLinkedApp(),
+      utils.getVersionChangelog(version),
+    ]))
+    .then(([app, changelog]) => {
       context.line(`Preparing to promote version ${version} of your app "${app.title}".\n`);
+
+      const body = {};
+
+      if (changelog) {
+        context.line(colors.green(`Changelog found for ${version}!`));
+        context.line(`\n---\n${changelog}\n---\n`);
+        body.changelog = changelog;
+      } else {
+        context.line(`${colors.yellow('Warning!')} Changelog not found. Please create a \`CHANGELOG.md\` file in a format similar to ${colors.cyan('https://github.com/zapier/zapier-platform-cli/blob/master/CHANGELOG.md')}, with user-facing descriptions.\n`);
+      }
+
```

Comment:
Does it mean when a changelog is absent, we still go ahead and promote the app right away? I think we should ask the user here if they want to continue without a changelog. And even if a changelog is found, we should also print the changelog and ask the user if they want to continue to give them a chance to edit the changelog.

Reply:
That's a great point! For now we don't have that concept of an interactive command so I didn't want to create that just for this, especially because eventually we'll make changelog required (in some future major version), and we'll start throwing an error when it's not found.

I hope that makes sense!

Reply:
Maybe I miss a context here. But don't we already have `zapier link` command, which asks the user which app to link?

Reply:
Oh 🤦‍♂️, you're right. I'll look into that, then!

Reply:
Alright, added!